app/assets/javascripts/spreadsheet.js

```diff
@@ -68,4 +43,40 @@ $(document).ready(function() {
 	    $(escape_characters(form_parent_id)).submit();
 	});
     });
+
+    // The bulk upload form can also be reached directly without opening a modal
+    if($("#spreadsheet-upload-container").length) {
+	load_upload_form();
+    }
 });
+
+
+function load_upload_form()
```

Comment:
It would be great if this could be encapsulated and not added to `global` scope.